osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -308,6 +311,26 @@ private void load(AudioManager audio, DialogOverlay dialog, OsuColour colours, S
                     }
                 });
             }
+
+            backgroundBlurBindable = config.GetBindable<float>(OsuSetting.MenuBlurLevel);
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            backgroundBlurBindable.ValueChanged += _ => updateBackgroundBlur();
+            updateBackgroundBlur(); // for first time opening beatmap selection
+        }
+
+        private void updateBackgroundBlur()
+        {
+            BackgroundBlur = backgroundBlurBindable.Value * 25;
```

Comment:
this 25 should still be a constant as it was. `BlurAmount` is already exposed from `ScreenWithBeatmapBackground` and can be used in each screen to apply different multiplications of it.